span.go

```diff
@@ -18,6 +18,10 @@ type spanImpl struct {
 	raw        RawSpan
 }
 
+var spanPool = &sync.Pool{New: func() interface{} {
```

Comment:
this makes me shed a silent tear, but it's not the first time a `sync.Pool` moved into a global :)